bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java

```diff
@@ -52,7 +59,10 @@
     public static final String SELECTOR = "cmp_download";
     public static final String INLINE_SELECTOR = "inline";
 
-    private static final String CONTENT_DISPOSITION = "Content-Disposition";
+    private static final String CONTENT_DISPOSITION_HEADER = "Content-Disposition";
+    private static final String RFC_DATE_FORMAT = "EEE, dd MMM yyyy HH:mm:ss zzz";
+    private static final String IF_MODIFIED_SINCE_HEADER = "If-Modified-Since";
```

Comment:
Duplicate of https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/org/apache/sling/api/servlets/HttpConstants.html#HEADER_IF_MODIFIED_SINCE